### PR TITLE
task/bugs_and_improvements/hebrewfy

### PR DESCRIPTION
**Pull request by teamagam5**
Hebrewfied!!!

### CODE REVIEW

File: app/src/main/res/layout/dialog_send_geo_message.xml [hunk 1]
```diff
@@ -19,11 +20,13 @@
             android:id="@+id/dialog_send_geo_message_text"
             android:layout_width="fill_parent"
             android:layout_height="wrap_content"
+            android:textAlignment="center"
             android:fontFamily="serif"
-            android:paddingLeft="20dp"
+            android:paddingStart="20dp"
```
Comment (teamagam3) [inline app/src/main/res/layout/dialog_send_geo_message.xml]: You can extract it to values.xml.
I'm not sure if it's necessary though.

File: app/src/main/res/layout/dialog_send_geo_message.xml [hunk 2]
```diff
@@ -32,21 +35,22 @@
...
+                android:layout_width="0dp"
+                android:layout_weight="1"
                 android:layout_height="wrap_content"
                 android:layout_gravity="start|center"
                 android:fontFamily="serif"
-                android:hint="Please describe the location..."
-                android:paddingLeft="20dp"
-                android:text="@={viewModel.text}" />
+                android:hint="@string/dialog_message_geo_text"
+                android:paddingStart="20dp"
```
Comment (teamagam3) [inline app/src/main/res/layout/dialog_send_geo_message.xml]: Same

File: app/src/main/res/layout/fragment_toolbar.xml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 <android.support.design.widget.AppBarLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/main.appbar"
-    android:layout_width="match_parent"
+                                            android:layout_width="match_parent"
```
Comment (teamagam3) [inline app/src/main/res/layout/fragment_toolbar.xml]: Fix indent

File: app/src/main/res/values-iw/strings.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (teamagam3) [inline app/src/main/res/values-iw/strings.xml:1]: Don't we want  to keep to old values?
What happens when the device requests English content? Will it give him the default strings.xml even if it's not in English?

File: app/src/main/res/layout/dialog_send_geo_message.xml (hunk 1 above)
Comment (BarAgam) [reply app/src/main/res/layout/dialog_send_geo_message.xml]: K

File: app/src/main/res/layout/dialog_send_geo_message.xml (hunk 2 above)
Comment (BarAgam) [reply app/src/main/res/layout/dialog_send_geo_message.xml]: K

File: app/src/main/res/layout/fragment_toolbar.xml (hunk 3 above)
Comment (BarAgam) [reply app/src/main/res/layout/fragment_toolbar.xml]: K

**Pull request merged**